src/ensembleIndex.js

```diff
@@ -10,12 +10,15 @@ export const ENSEMBLE_IMAGE_URL = (index) => {
 }
 
 export const getRoomAndTitleText = (ensembleIndex) => {
-  const wallTitle = ENSEMBLE[ensembleIndex].wallTitle;
+  const ensembleInfo = ENSEMBLE[ensembleIndex];
+  if (!ensembleInfo) return '';
```

Comment:
Ensemble index can be an empty string, in which case the resulting field will be undefined, causing a crash. This check prevents that and allows the objects to render properly.